### PR TITLE
Convert the error messages on the model list page into links.

### LINKED ISSUE #595: Make error links link to respective model details page
Screen Shot 2020-07-16 at 11 55 15 AM

### PR DESCRIPTION
**Pull request by hatched**
## Done

Convert the error messages on the model list page into links to the respective model details page.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Are the error links in the model details page links to the model details page?

## Details

Fixes #595

### CODE REVIEW

File: src/components/ModelTableList/StatusGroup.js [hunk 1]
```diff
@@ -40,14 +40,21 @@ function generateStatusTableHeaders(label, count) {
...
+  @param {String} activeUser The active user name.
   @return {Object} The react component for the warning message.
 */
-const generateWarningMessage = (model) => {
+const generateWarningMessage = (model, activeUser) => {
   const { messages } = getModelStatusGroupData(model);
   const title = messages.join("; ");
+  const link = generateModelDetailsLink(
+    model.model.name,
+    model.info && model.info.ownerTag,
```
Comment (barrymcgee) [inline src/components/ModelTableList/StatusGroup.js:51]: Optional chaining?

File: src/components/ModelTableList/_model-table-list.scss [hunk 2]
```diff
@@ -4,7 +4,8 @@
   margin-top: 0.5rem;
 }
 
-.model-table-list_error-message {
+.model-table-list_error-message a,
+.model-table-list_error-message a:visited {
```
Comment (barrymcgee) [inline src/components/ModelTableList/_model-table-list.scss:8]: You should use the parent (&) selector here to reprevent duplication;

```css
.model-table-list_error-message {
  &, &:hover { .. }
}
```

**Comment by barrymcgee** [comment]
From an accessibility point of view, rather than create two adjacent links to the same location, you wrap them in a single link.. other wise screenreaders and keyboard users have to double tab.

File: src/components/ModelTableList/_model-table-list.scss (hunk 2 above)
Comment (hatched) [reply src/components/ModelTableList/_model-table-list.scss:8]: done

**Comment by hatched** [comment]
The double tab concern is true, however the end goal for this is to link to the actual units/machines in error so they will ultimately end up being different links. That work will need to be completed once we finish the new unit/machine views however so this is somewhat of a stepping stone to that.

**Pull request merged**